src/utils/work/seekWorkResponse.js

```diff
@@ -28,21 +27,8 @@ const getRemainingWorkerStartTime = (creationTimestamp) => {
   return remainingTime + 60;
 };
 
-const seekFromS3 = async (ETag, experimentId, taskName) => {
-  let response;
-  try {
-    const url = `/v2/workResults/${experimentId}/${ETag}`;
-
-    response = await fetchAPI(url);
-  } catch (e) {
-    if (e.statusCode === httpStatusCodes.NOT_FOUND) {
-      return null;
-    }
-
-    throw e;
-  }
-
-  const storageResp = await fetch(response.signedUrl);
```

Comment:
We don't need the endpoint `workResults` anymore, the `signedUrl` (if that is what we receive) is returned through sockets.